changes/issue-2656-update-prometheus

```diff
@@ -0,0 +1,6 @@
+* Update to the latest version of the prometheus Go client library, resulting in some breaking changes in the exported metrics:
```

Comment:
Not sure if this is the right place to document the breaking changes, but the only place I've seen where metrics are discussed in the docs is in `docs/01-Using-Fleet/06-Monitoring-Fleet.md` and it doesn't mention any metrics name in details, just that we use prometheus format, so it seemed weird to introduce a note about specific changed metric names when there was no mention of them before.